### PR TITLE
feat(golden-signals-dashboard): Added a golden signals dashboard

### PR DESCRIPTION
**Pull request by zahra-siddiqa**
Added a dashobard to monitor all golden signals 

# Summary

Adding a dashboard to monitor all golden signals across APM, Infrastructure, Synthetics, Browser and Mobile. 

## Pre merge checklist

- [X] Did you check you NRQL syntax? - Does it work?
- [X] Did you include a Data source and Documentation reference?
- [X] Are all documentation links publicly accessible?
- [X] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [X] Did you check your descriptive content for spelling and grammar errors?
- [X] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [X] Does the PR contain a screenshot for each of your dashboards?
- [X] Do your screenshots show data?
- [X] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts - There are no alerts

- [x] Did you check that your alerts actually work?

### CODE REVIEW

File: quickstarts/golden-signals-dashboard/config.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+# Displayed in the UI (required)
+title: Golden signals for New Relic
+
+# Long-form description of the quickstart (required)
+description: |
+  Dashboard to monitor golden signals across APM, Infrastructure, Synthetics, Browser and Mobile with template variables to dynamically filter widgets.
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
+  Monitoring the golden signals allows you to quickly see an overview of the health of your application. This dashboard lets you monitor golden signals across APM, Infrastructure, Synthetics, Browser and Mobile with template variables to dynamically filter widgets
```
Comment (rahulbasu-nr) [inline quickstarts/golden-signals-dashboard/config.yml:13]:
```suggestion
  Monitoring the golden signals allows you to quickly see an overview of the health of your application. This dashboard lets you monitor golden signals across APM, Infrastructure, Synthetics, Browser and Mobile with template variables to dynamically filter widgets.
```

File: quickstarts/golden-signals-dashboard/config.yml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+#   - condition-a
+#   - condition-b
+
+# Reference to dashboards to be included in this quickstart
+dashboards:
+  - golden-signals-dashboard
+
+# Documentation references
+documentation:
+  - name: Instal APM agent
```
Comment (rahulbasu-nr) [inline quickstarts/golden-signals-dashboard/config.yml:42]:
```suggestion
  - name: Install APM agent
```

File: quickstarts/golden-signals-dashboard/config.yml [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    url: https://docs.newrelic.com/introduction-apm/
+    description: Monitor everything from the hundreds of dependencies of a modern stack down to simple web-transaction times and throughput of an app.
+  
+  - name: Instal Infrastructure agent
```
Comment (rahulbasu-nr) [inline quickstarts/golden-signals-dashboard/config.yml:46]:
```suggestion
  - name: Install Infrastructure agent
```

File: quickstarts/golden-signals-dashboard/config.yml [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    url: https://docs.newrelic.com/docs/infrastructure/install-infrastructure-agent/get-started/install-infrastructure-agent/
+    description: New Relic's infrastructure monitoring agent collects data about your hosts. It also reports data from some third party services, if enabled, and also log data.
+
+  - name: Set up Synthteics Monitors
```
Comment (rahulbasu-nr) [inline quickstarts/golden-signals-dashboard/config.yml:50]:
```suggestion
  - name: Set up Synthetics Monitors
```

File: quickstarts/golden-signals-dashboard/config.yml [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+    url: https://docs.newrelic.com/docs/synthetics/synthetic-monitoring/getting-started/get-started-synthetic-monitoring/
+    description: Synthetic monitoring helps you proactively catch and resolve issues before they affect your customers, all without an installation. 
+
+  - name: Instal Browser agent
```
Comment (rahulbasu-nr) [inline quickstarts/golden-signals-dashboard/config.yml:54]:
```suggestion
  - name: Install Browser agent
```

File: quickstarts/golden-signals-dashboard/config.yml [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+    url: https://docs.newrelic.com/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent/
+    description: The browser agent is a snippet of JavaScript code that monitors the performance of your app/site.     
+  
+  - name: Instal Mobile agent
```
Comment (rahulbasu-nr) [inline quickstarts/golden-signals-dashboard/config.yml:58]:
```suggestion
  - name: Install Mobile agent
```

File: quickstarts/golden-signals-dashboard/config.yml [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+authors:
+  - Zahra Siddiqa
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - golden signals
+  - monitoring
+  - latency
+  - traffic
+  - errors
```
Comment (rahulbasu-nr) [inline quickstarts/golden-signals-dashboard/config.yml:28]:
```suggestion
  - errors
  - error rates
```

**Review by rahulbasu-nr — CHANGES_REQUESTED** [review]
Please make these changes.

**Review by rahulbasu-nr — CHANGES_REQUESTED** [review]
Can you please make this change? Thanks.

**Review by rahulbasu-nr — DISMISSED** [review]
Looks good to me.

**Review by mdumpati — APPROVED** [review]
LGTM

File: quickstarts/golden-signals-dashboard/config.yml [hunk 8]
```diff
@@ -0,0 +1,65 @@
+# Sets the URL name of the quickstart on public I/O (required)
+slug: golden-signals-dashboard-for-New-Relic
```
Comment (aswanson-nr) [inline quickstarts/golden-signals-dashboard/config.yml:2]: The slug needs to be lower case:
```suggestion
slug: golden-signals-dashboard-for-new-relic
```

**Review by aswanson-nr — CHANGES_REQUESTED** [review]
Hey @zahra-siddiqa, this looks great! Just saw one problem with the quickstart slug. 
Also, could you sign the CLA? We need that to be signed before we can merge.

**Comment by rahulbasu-nr** [comment]
Hello Zahra! I'm having difficulty locating a data source for this PR. @aswanson-nr, could you kindly advise if it's ready to proceed?

**Comment by zahra-siddiqa** [comment]
> Hello @zahra-siddiqa ! I'm having difficulty locating a data source for this PR. @aswanson-nr, could you kindly advise if it's ready to proceed?

Hi @rahulbasu-nr, the datasource for this dashboard is the data coming from New Relic APM, Infrastructure, Synthetics, Browser and Mobile. Do you need me to add this in the quickstart config file?

**Comment by rahulbasu-nr** [comment]
Yes Zahra! datasourceIds needs to be added to the config.yml file. Kindly follow this file as a [reference](https://github.com/newrelic/newrelic-quickstarts/blob/main/quickstarts/mean/config.yml).

**Review by sarahkitten — APPROVED** [review]

**Pull request merged**